src/events.ts

```diff
@@ -0,0 +1,24 @@
+export const techEvents = [
+  {
+    title: "May, 2023: Coffee.js",
+    date: "SAT, MAY, 13, 2023, 12:00 PM PDT",
+    location: "The Golden Horn Turkish Bakery & Cafe",
+    description:
+      "Join us a regular monthly social for JavaScript enthusiasts. Whether you're a seasoned JavaScript developer or just starting out, Coffee.js is the perfect opportunity to network with like-minded individuals, share your ideas and learn from others.",
+    imageUrl:
+      "https://secure.meetupstatic.com/photos/event/c/7/2/3/clean_511430979.jpeg",
+    website:
+      "https://www.meetup.com/vancouver-javascript-developers/events/292653989/",
+  },
+  // {
+  //   title: "May, 2023: Coffee.js",
```

Comment:
@devjoylee You can delete comment outs if you're not going to use them in the future, but if not, just ignore this comment:pray: